packages/components/src/styles/common-styles.spec.ts

```diff
@@ -100,3 +109,14 @@ describe('getBackfaceVisibilityJssStyle()', () => {
     expect(getBackfaceVisibilityJssStyle()).toMatchSnapshot();
   });
 });
+
+describe('getFrostedGlassBackgroundJssStyles()', () => {
+  it.each<Parameters<typeof getFrostedGlassBackgroundJssStyles>>([
+    [true, 'motionDurationShort', 'light'],
+    [false, 'motionDurationModerate', 'dark'],
+    [true, 'motionDurationLong', 'light'],
+    [false, 'motionDurationVeryLong', 'dark'],
+  ])('should return correct JssStyle for isVisible: %s, duration: %s and theme: %s', (isVisible, duration, theme) => {
```

Comment:
See previous comment. Here snapshots make sense since it is too much to manually maintain.